README.md

```diff
@@ -115,18 +115,18 @@ There you will find Express.js, TypeScript and Websocket examples.
 
 ## Supported specification features
 
-| Core Specification |  [v0.3](v03spec) | [v1.0](v1spec) |
+| Core Specification |  [v0.3](https://github.com/cloudevents/spec/tree/v0.3) | [v1.0](https://github.com/cloudevents/spec/tree/v1.0) |
```

Comment:
Can these links be to the actual spec document instead of just to the top level of the repo? The others make sense as top-level since they are about multiple specs, but this is the primary spec source.

Reply:
sure

Reply:
@lance updated